app/javascript/components/DonationBands/__snapshots__/DonationBands.test.js.snap

```diff
@@ -36,12 +36,14 @@ exports[`renders all amounts with the currency symbol 1`] = `
       "messages": Object {
         "basics.or": "or",
         "branding.description": "This is who we are.",
+        "call_tool.errors.akid": "Sorry, but for safety reasons, the calling tool is limited to recognized members who enter the website through a campaign email.",
```

Comment:
I'd remove _recognized_ from the sentence so that it just says _"limited to members who enter [...] through a campaign email"_